LICENSE.txt is placed separately to /usr/share/licenses.
There is no need to duplicate it to /usr/share.

--- a/gdal.cmake
+++ b/gdal.cmake
@@ -576,7 +576,6 @@
   PROPERTY PUBLIC_HEADER ${CMAKE_CURRENT_BINARY_DIR}/port/cpl_config.h)
 
 set(GDAL_DATA_FILES
-    LICENSE.TXT
     data/GDALLogoBW.svg
     data/GDALLogoColor.svg
     data/GDALLogoGS.svg
